integration_tests/models/sql/test_unpivot.sql

```diff
@@ -11,19 +11,15 @@
     {% set exclude = ['customer_id', 'created_at'] %}
 {% endif %}
 
-select
-    customer_id,
-    created_at,
-    case
-        when '{{ target.name }}' = 'snowflake' then lower(field_name)
-        else field_name
-    end as field_name,
-    value
+select *
```

Comment:
looks like the test on this model is failing on Snowflake:
<img width="879" alt="Screen Shot 2019-06-20 at 11 56 52 AM" src="https://user-images.githubusercontent.com/1541139/59863481-c9938380-9352-11e9-973c-796b2613d1a4.png">

Can you copy in the hack from `integration_tests/models/sql/test_unpivot_original_api.sql` here?

Maybe something like:
```sql
select
    customer_id,
    created_at,
    case
        when '{{ target.name }}' = 'snowflake' then lower(prop)
        else prop
    end as prop,
    value
```

This will just lowercase the `prop` column of the `unpivot` macro so that the seed files will match up in the equality test. Lmk if you want me to help out with that - i know it's hard to test snowflake-specific functionality without access to a snowflake database!

Reply:
thanks for the help @drewbanin. I think I've fixed it but I still cant get the CI working. Can you re-trigger please?

Reply:
hmm, this still doesn't look right to me (and the tests are still failing!)

can you try using the logic i sent over above in `test_unpivot.sql`?
```
select
    customer_id,
    created_at,
    case
        when '{{ target.name }}' = 'snowflake' then lower(prop)
        else prop
    end as prop,
    value
```

We need to lowercase the `prop` column to get the tests to pass on Snowflake